charts/brigade/templates/apiserver/ingress.yaml

```diff
@@ -19,13 +19,17 @@ spec:
   - host: {{ .Values.apiserver.host }}
     http:
       paths:
-      - backend:
-          serviceName: {{ include "brigade.apiserver.fullname" . }}
-          {{- if .Values.apiserver.tls.enabled }}
-          servicePort: 443
-          {{- else }}
-          servicePort: 80
-          {{- end }}
+      - pathType: ImplementationSpecific
+        path: /
+        backend:
+          service:
+            name: {{ include "brigade.apiserver.fullname" . }}
+            port:
+              {{- if .Values.apiserver.tls.enabled }}
+              number: 443
+              {{- else }}
+              number: 80
+              {{- end }}
```

Comment:
Does this new schema work even with the v1beta1 api?

Reply:
Oh right, I didn't think to try, but I assume not.  Shall we commit to the new api?  Or I suppose I can update the template to support both by toggling how the schema looks depending on version?

Reply:
Straightforward enough to toggle the schema, so I've done so.  Will test against a 1.18.x cluster as well...

Reply:
Looks good in my testing.